dev/DynamicDependency/M.AM.DD.CreatePackageDependencyOptions.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "Microsoft.ApplicationModel.DynamicDependency.CreatePackageDependencyOptions.g.h"
+
+#include "winrt_namespaces.h"
+
+namespace winrt::Microsoft::ApplicationModel::DynamicDependency::implementation
+{
+    struct CreatePackageDependencyOptions : CreatePackageDependencyOptionsT<CreatePackageDependencyOptions>
+    {
+        CreatePackageDependencyOptions() = default;
+
+        winrt::PackageDependencyProcessorArchitectures Architectures();
+        void Architectures(Microsoft::ApplicationModel::DynamicDependency::PackageDependencyProcessorArchitectures const& value);
+        bool VerifyDependencyResolution();
+        void VerifyDependencyResolution(bool value);
+        winrt::PackageDependencyLifetimeArtifactKind LifetimeArtifactKind();
+        void LifetimeArtifactKind(Microsoft::ApplicationModel::DynamicDependency::PackageDependencyLifetimeArtifactKind const& value);
+        hstring LifetimeArtifact();
+        void LifetimeArtifact(hstring const& value);
+
+    private:
+        winrt::PackageDependencyProcessorArchitectures m_architectures = winrt::PackageDependencyProcessorArchitectures::None;
+        bool m_verifyDependencyResolution = false;
```

Comment:
Prefer the initializer syntax for in-class initializers (see earlier comment)

Reply:
Sure. I'll do a pass over the whole thing (while the code's still small). If I miss anything I can take care of it in the next PR (this isn't the end)